source/Api/Client/VirtualApi/Fido.Api.Client.VirtualApi.Abstract.pas

```diff
@@ -715,10 +714,10 @@ class function TAbstractClientVirtualApi<T, IConfiguration>.InspectMethod(const
   Content := '';
   HandleRedirects := true;
 
-  QueryParams := TCollections.CreateDictionary<string, string>(TIStringComparer.Ordinal);
-  HeaderParams := TCollections.CreateDictionary<string, string>(TIStringComparer.Ordinal);
-  FormParams := TCollections.CreateDictionary<string, string>(TIStringComparer.Ordinal);
-  FileParams := TCollections.CreateDictionary<string, string>(TIStringComparer.Ordinal);
+  QueryParams := TCollections.CreateSortedDictionary<string, string>(TIStringComparer.Ordinal as IComparer<string>);
```

Comment:
Why the switch to a sorted dictionary? Just curious

Reply:
Because when I iterate the keys I need them sorted by insertion order.